micro-ui/web/micro-ui-internals/packages/modules/workbench/README.md

```
@@ -79,6 +79,8 @@ _Add this configuration to enable this module [MDMS Enabling Workbench Module](h
 
 6 . Data push for any API based on schema
 
+7 . Json-edit-react to view and edit schema 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)